core/gui/src/app/workspace/component/menu/menu.component.html

```diff
@@ -40,7 +40,8 @@
           nz-button
           nzType="primary"
           [disabled]="!workflowVersionService.modificationEnabledBeforeTempWorkflow"
-          (click)="revertToVersion()">
+          (click)="revertToVersion()"
+          style="width: 160px">
```

Comment:
try not to hard code it? it may not work for different resolution and browsers.

Reply:
ask @aglinxinyuan regarding this?

Reply:
It's okay to have buttons of a fixed size. Since we have fixed font sizes, this should work for different resolutions and browsers.